### ISSUE TITLE
Bower is prompting to update

### ISSUE THREAD

**Issue by hdgarrood** [purescript-contrib/pulp#114]
```
$ pulp init
* Generating project skeleton in /tmp/pulp-lol
bower cached        git://github.com/purescript/purescript-console.git#0.1.1
bower validate      0.1.1 against git://github.com/purescript/purescript-console.git#^0.1.0

<snip>

┌──────────────────────────────────────────┐
│ Update available: 1.6.6 (current: 1.6.5) │
│ Run npm install -g bower to update.      │
└──────────────────────────────────────────┘
```

Of course, `npm install -g bower` doesn't make this message go away, because Pulp is using its own Bower which was installed as a dependency. So this message is quite confusing. Especially if you try `bower --version` afterwards and that bower reports a different version!

To avoid confusion, I propose that we try to avoid this message from being shown. Hopefully bower has some command line flag for this.

**Comment by bodil** [comment]
Bower does not, which should surprise no one. :)

The best way to solve this would be for `pulp` to be able to update its dependency. All other solutions (using global `bower` if available, suppressing the message) seem poor to me. Not sure how to proceed here.

**Comment by hdgarrood** [comment]
Resolved by #141.
